.travis.yml

```diff
@@ -18,6 +18,7 @@
 # TravisCI is being used as an alternative to GitHub Actions until
 # the latter provides support for ARM64 CPU architecture (PARQUET-1980)
 
+dist: focal
 distro: bionic
```

Comment:
Sorry, I didn't notice it earlier - you need to remove `distro: bionic`. 
`distro` is a typo anyway, so it was falling back to the default (`bionic`) until now.

Reply:
Thanks, @martin-g. I didn't realize that `dist` is for distro. :)